bonita-home/src/main/resources/client/platform/work/i18n/portal-js.po

```diff
@@ -17,273 +17,1092 @@ msgstr ""
+msgid "Start date"
 msgstr ""
 
-msgid  "Failed instances"
+#: main/features/admin/cases/list/cases-list.html
+msgid "Start for"
 msgstr ""
 
-msgid  "Executing, completing, initializing instances"
+msgid "Started"
 msgstr ""
 
-msgid  "Ready, waiting instances"
+#: main/features/admin/cases/list/cases-list.html
+msgid "Started by"
 msgstr ""
 
-msgid  "Items per page"
+#: main/features/admin/monitoring/monitoring.html
+#: main/features/admin/cases/list/cases-list.html
+msgid "State"
 msgstr ""
 
-msgid  "Columns selection"
+#: main/features/admin/applications/import-application.html
+msgid "Status of import"
 msgstr ""
 
-msgid  "Table settings"
+msgid "Successfully updated categories"
 msgstr ""
 
-msgid  "sort by ID ascending"
+msgid "Table settings"
 msgstr ""
 
-msgid  "sort by ID descending"
+msgid "Target is not set"
 msgstr ""
 
-msgid  "sort by Process name ascending"
+#: main/features/admin/processes/details/forms.html
+msgid "Task name"
 msgstr ""
 
-msgid  "sort by Process name descending"
+#: main/features/admin/processVisu/visuShowKey.html
+msgid "Task1"
 msgstr ""
 
-msgid  "sort by Start date ascending"
+msgid "Technical error."
 msgstr ""
 
-msgid  "sort by Start date descending"
+#: main/features/admin/processes/details/menu.html
+msgid "The Process cannot be enabled"
 msgstr ""
 
-msgid  "sort by End date ascending"
+#: main/features/admin/applications/delete-application.html
+msgid "The application"
+msgstr ""
+
+#: main/features/admin/processes/details/forms.html
+msgid "The current process has no task defined."
+msgstr ""
+
+#: main/features/admin/processes/details/process-connectors.html
+msgid "The current process has no connectors defined."
+msgstr ""
+
+#: main/features/admin/processes/details/params.html
+msgid "The current process has no parameters defined."
+msgstr ""
+
+#: main/features/admin/cases/list/cases-list-deletion-modal.html
+msgid ""
+"The deleted case will be permanently deleted and will not be stored in the "
+"archives."
+msgstr ""
+
+#: main/features/admin/cases/list/cases-list-deletion-modal.html
+msgid "The deleted case will be permanently deleted from the archives."
+msgstr ""
+
+#: main/features/admin/cases/list/cases-list-deletion-modal.html
+msgid ""
+"These deleted cases will be permanently deleted and will not be stored in "
+"the archives."
+msgstr ""
+
+#: main/features/admin/cases/list/cases-list-deletion-modal.html
+msgid "These deleted cases will be permanently deleted from the archives."
+msgstr ""
+
+#: main/features/admin/applications/applications-list.html
+msgid "To create an application, click New or Import."
+msgstr ""
+
+#: main/features/admin/processes/details/process-managers-mapping.html
+#: main/features/admin/processes/details/actors-mapping.html
+msgid "To do so, go to"
+msgstr ""
+
+msgid ""
+"To use this feature, create a new version of the process in Bonita BPM "
+"Studio version 6.4 and above, then create and install the new .bar file in "
```

Comment:
Bonita BPM Studio should externalized in order to be able to override it for OEM